examples/data-generator/src/reviews.ts

```diff
@@ -23,15 +23,18 @@ export default (db, { serializeDate }) => {
                     .filter(() => weightedBoolean(40)) // reviewers review 40% of their products
                     .map(product => {
                         const date = randomDate(command.date);
-                        const status = isAfter(aMonthAgo, date)
-                            ? weightedArrayElement(
-                                  ['accepted', 'rejected'],
-                                  [3, 1]
-                              )
-                            : weightedArrayElement(
-                                  ['pending', 'accepted', 'rejected'],
-                                  [5, 3, 1]
-                              );
+                        const status =
+                            isAfter(aMonthAgo, date) ?
+                                weightedArrayElement(
+                                    ['accepted', 'rejected'],
+                                    [3, 1]
+                                )
+                            : (
+                                weightedArrayElement(
+                                    ['pending', 'accepted', 'rejected'],
+                                    [5, 3, 1]
+                                )
+                            );
```

Comment:
Why is there an extra paren here, looks weird.

Reply:
Equal indentation is used to show that we have two equal alternatives here. Does that really look that bad?

Reply:
The parens here are now eliminated